WordPress/Sniffs/WhiteSpace/OperatorSpacingSniff.php

```diff
@@ -17,194 +21,39 @@
  * @package WPCS\WordPressCodingStandards
  *
  * @since   0.1.0
- * @since   0.3.0 This sniff now has the ability to fix the issues it flags.
- * @since   0.12.0 This class now extends WordPress_Sniff.
+ * @since   0.3.0  This sniff now has the ability to fix the issues it flags.
+ * @since   0.12.0 This sniff used to be a copy of a very old and outdated version of the
+ *                 upstream sniff.
+ *                 Now, the sniff defers completely to the upstream sniff, adding just one
+ *                 additional token - T_BOOLEAN_NOT - via the registration method
+ *                 and changing the value of the customizable $ignoreNewlines property.
  *
- * Last synced with base class December 2008 at commit f01746fd1c89e98174b16c76efd325825eb58bf1.
```

Comment:
LOL, yup, that was when the original was cloned and never synced again ;-)

Until now that is ;-)